artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +94,51 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	artVersion, err := servicesManager.GetVersion()
+	if err != nil {
+		return err
+	}
+	artVersionToStr, err := strconv.Atoi(artVersion[0:1])
+	if err != nil {
+		return err
+	}
+	log.Info(bpc.printBuildInfoLink(artVersionToStr, buildTime))
+
 	if !bpc.config.DryRun {
 		return utils.RemoveBuildDir(bpc.buildConfiguration.BuildName, bpc.buildConfiguration.BuildNumber, bpc.buildConfiguration.Project)
 	}
 	return nil
 }
 
+func (bpc *BuildPublishCommand) printBuildInfoLink(artVersion int, buildTime time.Time) string {
```

Comment:
The name of this method is wrong, beciuase the method doe3s not print anything. In addition, nothing that the following string is duplicate inside this method - we should avoid this as well - 
```Build info successfully deployed. Browse it in Artifactory under```.
Here's what I suggest:
1. Let's rename this method to ```getBuildInfoUiUrl```.
2. The method will return the URL only, without the ```Build info successfully deployed. Browse it in Artifactory under``` string.